HCaptcha/Classes/HCaptcha.swift

```diff
@@ -139,9 +161,16 @@ public class HCaptcha {
     public convenience init(
         apiKey: String? = nil,
         baseURL: URL? = nil,
-        endpoint: Endpoint = .default,
         locale: Locale? = nil,
-        size: Size = .invisible
+        size: Size = .invisible,
+        rqdata: String? = nil,
+        sentry: Bool = false,
+        apiEndpoint: URL = URL(string: "https://hcaptcha.com/1/api.js")!,
+        endpoint: URL = URL(string: "https://hcaptcha.com")!,
+        reportapi: URL = URL(string: "https://accounts.hcaptcha.com")!,
+        assethost: URL = URL(string: "https://assets.hcaptcha.com")!,
```

Comment:
```suggestion
        assethost: URL = URL(string: "https://newassets.hcaptcha.com")!,
```

But you shouldn't be passing in these optional values unless they're specified in init.

Reply:
Done (for tests)